source/options.js

```diff
@@ -1,6 +1,9 @@
 // Don't forget to import this wherever you use it
 import browser from 'webextension-polyfill';
 
+import 'webext-base-css/webext-base.css';
```

Comment:
This should work already because the package’s `main` field points to it. If it doesn’t work, it can stay. 

```suggestion
import 'webext-base-css';
```

Reply:
Yep, this seems to work